src/utils/command_handlers.py

```diff
@@ -67,3 +68,25 @@ def handle_search_contact(manager: ContactManager) -> None:
             print(f"An error occurred during the search: {ex}")
     else:
         print("Invalid search type. Please choose 'name', 'email', or 'phone'.")
+
+
+def handle_command(self, command: str) -> None:
```

Comment:
@AndreyShvachko hello!
In general, this is correct, but there is only one point: the handle_command function is unnecessary here. Its work is performed at the main.py level
That is, we can immediately call the display_all_contacts function from there.